src/main/scala/abalone/variant/GrandAbalone.scala

```diff
@@ -34,10 +34,10 @@ case object GrandAbalone
   /** The sequence of the number of actions per turn is 12* (P1 plays one move, then, starting with P2, both
     * players have two actions per turn).
     */
-  override def isAutoEndTurn(sit: Situation, orig: Pos, dest: Pos): Boolean =
-    sit.board.history.prevPlayer.fold(true)(p => p == sit.player)
+  override def isAutoEndTurn(situation: Situation, orig: Pos, dest: Pos): Boolean =
+    !situation.board.history.currentTurn.isEmpty || situation.board.history.lastTurn.isEmpty
 
   /** Belgian daisy. */
   override def initialFen: FEN =
-    format.FEN("SS2ss/SSS1sss/1SS2ss1/9/ss6SS/sss5SSS/ss6SS/9/1SS2ss1/SSS1sss/SS2ss 0 0 b * 0 1")
+    format.FEN("SS2ss/SSS1sss/1SS2ss1/9/ss6SS/sss5SSS/ss6SS/9/1SS2ss1/SSS1sss/SS2ss 0 0 b 0 1 - -")
```

Comment:
Do the pieces need to be flipped as your FEN format changed (like you just did with the Abalone `initialFen`?
I would remove the final two `- -` here.